samples/excel/85-preview-apis/range-auto-fill.yaml

```diff
@@ -0,0 +1,156 @@
+            sumCell.format.autofitColumns();
+            await context.sync();
+          });
+        }
+
+        async function setupData() {
+          await Excel.run(async (context) => {
+            context.workbook.worksheets.getItemOrNullObject("Sample").delete();
+            const sheet = context.workbook.worksheets.add("Sample");
+
+            const data = [
+              ["Fruit", "Q1", "Q2", "Q3", "Q4", 2017, "Q1", "Q2", "Q3", "Q4", 2018, "Q1", "Q2", "Q3", "Q4", 2019],
+              ["Quinces", 500, 2000, 5000, 1500, , 600, 2200, 5400, 1300, , 300, 1500, 4300, 1200, ""],
+              ["Plums", 700, 900, 1300, 800, , 800, 1000, 2000, 700, , 300, 600, 1100, 700, ""],
+              ["Apples", 8000, 3000, 4500, 7500, , 8900, 4000, 3500, 4500, , 6000, 2500, 4500, 9500, ""],
+              ["Peaches", 12000, 7000, 8000, 13000, , 16000, 10000, 8500, 12000, , 11000, 5600, 6500, 11000, ""]
+            ];
+
+            sheet.getRange("A3:P7").values = data;
+            sheet.getRange("A1").values = [["Fruit Sales"]];
+            sheet.getRange("A1").style = "Heading1";
+            sheet.getRanges("B2:P3,A3:A7").format.font.bold = true;
+
+            sheet.getRange("A1:P7").format.autofitColumns();
+            sheet.activate();
+
+            await context.sync();
+          });
+        }
+
+        async function setupFirstSums() {
+          await Excel.run(async (context) => {
+            const sheet = context.workbook.worksheets.getActiveWorksheet();
+            const sumRange = sheet.getRange("F4");
+            sumRange.formulas = [["=SUM(B4:E4)"]];
+            sumRange.format.fill.color = "LightBlue";
+            sumRange.format.font.bold = true;
+            sheet.getRanges("K4,P4").copyFrom("F4", Excel.RangeCopyType.all);
+            sheet.getRanges("F4,K4,P4").format.autofitColumns();
+            await context.sync();
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback, option?) {
+          try {
+            await callback(option);
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to provide cell data for surrounding cells using auto fill.</p>
+        </section>
+
+        <section class="setup ms-font-m">
+          <h3>Set up</h3>
+          <button id="setup-data" class="ms-Button">
+            <span class="ms-Button-label">Add sample data</span>
+        </button>
+          <p />
+          <button id="setup-first-sums" class="ms-Button">
+            <span class="ms-Button-label">Add first sum cells</span>
+        </button>
+        </section>
+
+        <section class="samples ms-font-m">
```

Comment:
Recommend shortening (rewording) each of these 3 button labels, if possible. Maybe something along these lines instead?
- `Auto fill values in column F`
- `Auto fill formats in column K`
- `Auto fill everything in column P`

As buttons are currently labeled, the length of each button label is too long to fit within the button, unless I resize my "Run" pane to be very wide (which'll hinder usability of the sample for folks using smaller screens -- since the "Run" pane having to be so wide will make them have to manually scroll side to side to see all of the data in the worksheet).

![image](https://user-images.githubusercontent.com/3019529/65358983-edfc3780-dbaf-11e9-872d-9cd0ab63452d.png)